releasenotes/notes/remove-legacy-operator-88e8762cf0156d40.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    Remove legacy operator in :mod:`qiskit.opflow.legacy`.
+  - |
+    Add a missing :meth:`qiskit.opflow.operator_base.OperatorBase.to_spmatrix`.
```

Comment:
A release note isn't actually needed here. There isn't any upgrade implication on terra, `qiskit.opflow.legacy` was never a released module. There is an implication qiskit-aqua if/when the legacy path gets removed there, but that should be handled in the aqua's release notes.

As a general rule individual changes made to qiskit.algorithms, or qiskit.opflow in terra don't need separate release notes before the 0.17.0 release. We'll have a single release note that says we've added all the things and link to the migration guide we'll write for #5493

Reply:
Thank for your information. I remove it.